eth2near/ethashproof/ethash/consensus.go

```diff
@@ -570,14 +572,14 @@ func (ethash *Ethash) Finalize(chain consensus.ChainReader, header *types.Header
 	header.Root = state.IntermediateRoot(chain.Config().IsEIP158(header.Number))
 
 	// Header seems complete, assemble into a block and return
-	return types.NewBlock(header, txs, uncles, receipts), nil
+	return types.NewBlock(header, txs, uncles, receipts, trie.NewStackTrie(nil)), nil
```

Comment:
why do we add `trie.NewStackTrie(nil)` here?

Reply:
We are using a newer version of go-ethereum and this is a new required argument.